lib/modules/manager/helmv3/artifacts.spec.ts

```diff
@@ -35,12 +41,29 @@ const ociLockFile1Alias = Fixtures.get('oci_1_alias.lock');
 const ociLockFile2Alias = Fixtures.get('oci_2_alias.lock');
 const chartFileAlias = Fixtures.get('ChartAlias.yaml');
 
+const ociLockFile1ECR = Fixtures.get('oci_1_ecr.lock');
+const ociLockFile2ECR = Fixtures.get('oci_2_ecr.lock');
```

Comment:
I think we don't need those, just return a simple string, they're not validated.

Reply:
I tried changing these to strings, but it turns out that at least the original lockfile must be parsed by something, because I get an error like the following:

```
  ● modules/manager/helmv3/artifacts › supports ECR authentication

    expect(received).toMatchObject(expected)

    - Expected  - 4
    + Received  + 3

      Array [
        Object {
    -     "file": Object {
    -       "contents": "<Chart.lock after>",
    -       "path": "Chart.lock",
    -       "type": "addition",
    +     "artifactError": Object {
    +       "lockFile": "Chart.lock",
    +       "stderr": "Cannot read properties of undefined (reading 'repository')",
          },
        },
      ]

      780 |         },
      781 |       })
    > 782 |     ).toMatchObject([
          |       ^
      783 |       {
      784 |         file: {
      785 |           type: 'addition',

      at Object.<anonymous> (lib/modules/manager/helmv3/artifacts.spec.ts:782:7)
```